README.md

```diff
@@ -13,6 +13,7 @@ High-performance inference of [OpenAI's Whisper](https://github.com/openai/whisp
 - Runs on the CPU
 - [C-style API](https://github.com/ggerganov/whisper.cpp/blob/master/whisper.h)
 - Supported platforms: Linux, Mac OS (Intel and Arm), Windows (MinGW), Raspberry Pi, Android
+- Can output the transcription/translation to a text file or subtitle file (VTT or SRT formats)
```

Comment:
I want to keep this section with features only about the library. The subtitle generation is part of the `main` example and not worth mentioning here.

```suggestion
```

The rest of the changes are good.